Gallery/ImagesList/ImagesListViewController.swift

```diff
@@ -4,40 +4,58 @@ class ImagesListViewController: UIViewController {
     
     @IBOutlet private var tableView: UITableView!
     
-    private var photosName = [String]()
+    private let ShowSingleImageSegueID = "ShowSingleImage"
+    private var photoNames = [String]()
     
     override func viewDidLoad() {
+        super.viewDidLoad()
         tableView.delegate = self
         tableView.dataSource = self
-        photosName = Array(0..<20).map{ "\($0)" }
+        photoNames = Array(0..<20).map{ "\($0)" }
+    }
+    
+    // Utility
+    
+    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
+        
+        if segue.identifier == ShowSingleImageSegueID {
+            let viewController = segue.destination as! SingleImageViewController
+            let indexPath = sender as! IndexPath
```

Comment:
давай на безопасные операции этот код перепишем
```suggestion
            let viewController = segue.destination as! SingleImageViewController
            let indexPath = sender as! IndexPath
```